parlai/core/dict.py

```diff
@@ -324,6 +324,12 @@ def __init__(self, opt: Opt, shared=None):
             if opt.get('dict_file'):
                 self.save_path = opt['dict_file']
 
+    def is_prebuilt(self):
```

Comment:
hmm so why not just return False if this is overridden by the Gpt2 dictionary?

also, there are other HF tokenizers out there in ParlAI (that Dexter wrote); these should probably return False too, no?

Reply:
This is a misnomer. There's the HuggingFace GPT2 dictionary, and there's the gpt-2 tokenizer, and they're different.